### ISSUE TITLE
Temporarily use scriptableorg/NimbusSans until Artifex Software fixes it

### ISSUE THREAD

**Issue by louies0623** [helloSystem/hello#539]
https://github.com/ArtifexSoftware/urw-base35-fonts/issues/25#issue-330490089 This issue has not been resolved and there is no sign of starting a fix, alternatives must be found until they start fixing.

Scriptableorg/NimbusSans
https://github.com/scriptableorg/NimbusSans

The one on the right is the latest version of Artifex Software 2020.9.10, and the one on the left is scriptableorg NimbusSans (May 9, 2020).
![螢幕快照 2023-09-06 上午5 07 01](https://github.com/helloSystem/hello/assets/44593430/f63a7329-6483-45ab-bbbf-9e69d295eba4)

**Comment by probonopd** [comment]
We have already changed our theme to work with URW Nimbus Sans, so I think it's ok now?

**Comment by louies0623** [comment]
No.

**Comment by louies0623** [comment]
No, this will not work at all in languages ​​other than those supported by URW Nimbus Sans.

**Comment by probonopd** [comment]
We have adjusted everything to look good with URW Nimbus Sans. But that means that all other fonts look wrong. So maybe we need to get URW Nimbus Sans fixed indeed.
